app/src/main/java/lt/vilnius/tvarkau/fragments/MultipleProblemsMapFragment.kt

```diff
@@ -54,7 +58,7 @@ class MultipleProblemsMapFragment : BaseMapFragment(),
             Timber.e(throwable)
         }
 
-        legacyApiService.getProblems(request)
+        subscription = legacyApiService.getProblems(request)
```

Comment:
I would prefer current style as you see right away if it's assigned to subscription or not.